Makefile.all.bak

```diff
@@ -0,0 +1,236 @@
+DESTDIR=/usr/local
```

Comment:
Please don't commit the `.bak` files. (I'm refraining from starting the CI because this might confuse it.)

Reply:
Of course! My apologies for the inconvenience!